circuit_knitting_toolbox/entanglement_forging/entanglement_forging_knitter.py

```diff
@@ -669,11 +666,11 @@ def _estimate_expvals(
         - session_id (str): The session id to use when calling primitive programs
 
     Returns:
-        - (Tuple[List[NDArray], List[NDArray], Optional[str]]): the expectation values for the
+        - (Tuple[List[np.ndarray], List[np.ndarray], Optional[str]]): the expectation values for the
             tensor circuits and superposition circuits
     """
-    all_circuits = tensor_ansatze + superposition_ansatze
```

Comment:
Deprecated usage. Don't need these as input for instantiating estimator anymore. Can pass in circuits during `run`